bigtable_input.py

```diff
@@ -467,27 +485,26 @@ def update_move_counts(self, start_game, end_game, interval=1000):
                 print('  games/sec:', len(h)/elapsed)
 
 
-def set_fresh_watermark(games, window_size, fresh_fraction=0.05, minimum_fresh=20000):
+def set_fresh_watermark(game_queue, latest_game, window_size, fresh_fraction=0.05, minimum_fresh=20000):
```

Comment:
The words "fresh watermark" and "high water mark" are both used here. "High water mark" has an association of highest seen, rather than requirement. Maybe `set_freshwater_mark` or `set_fresh_game_mark`?